lib/Exceptions/ArtistHasArtworkException.php

```diff
@@ -0,0 +1,7 @@
+<?
+namespace Exceptions;
+
+class ArtistHasArtworkException extends AppException{
+	/** @var string $message */
+	protected $message = 'Artist has artwork assoicated with it. Can’t delete.';
```

Comment:
Spelling error, also let's make this a complete sentence

Reply:
I took another whack at rewriting the error message.